scss/ui/_island.scss

```diff
@@ -3,28 +3,42 @@
 //------------------------------------//
 
 // Harry Robert's "island"
-.island,
 .islet,
+.island,
+.island--dubs,
 .landmark,
+.landmark--dubs,
 .landmarklet {
   display: block;
   @include clearfix;
 
   > :last-child { margin-bottom: 0; }
 }
 
+.islet {
+  @include padding-vertical($half-spacing-unit);
+}
+
 .island {
   @include padding-vertical($base-spacing-unit);
 }
 
-.islet {
-  @include padding-vertical($half-spacing-unit);
+.island--dubs {
+  @include padding-vertical($base-spacing-unit * 2);
 }
 
 .landmark {
-  margin-bottom: 2 * $base-spacing-unit;
+  margin-bottom: $base-spacing-unit;
+}
+
+.landmark--dubs {
+  margin-bottom: $base-spacing-unit * 2;
 }
 
 .landmarklet {
   margin-bottom: $base-spacing-unit;
```

Comment:
If `.landmark` is now just `$base-spacing-unit`, should `.landmarklet` be `$half-spacing-unit`?

Reply:
Good catch! IMO, the `*let` classes (`island`, `landmarklet`) should have the same spacing.

Reply:
yes, great catch!